code/modules/suppressions/_core_suppression.dm

```diff
@@ -22,5 +23,6 @@
 /datum/suppression/proc/End()
 	priority_announce(end_text, name, sound=end_sound)
 	SSlobotomy_corp.core_suppression = null
+	SSlobotomy_corp.core_suppression_state = 2 // Finished core suppression
```

Comment:
```suggestion
	SSlobotomy_corp.core_suppression_state = max(SSlobotomy_corp.core_suppression_state, 2) // Finished core suppression
```
Before I forget about it. Right now it will overwrite Claw's death(3) state, so I gotta fix it.